lib/timeline/fps.dart

```diff
@@ -89,31 +89,45 @@ class FramesChart extends LineChart<FramesTracker> {
 }
 
 class FramesTracker {
```

Comment:
Now would be a good time to tweak this code enough so we can add tests ideally ones that run just on the VM so they are easy to write and this code is easy to port to hummingbird in the future.

Reply:
Done.